docs/operator-notes.md

```diff
@@ -62,6 +62,14 @@ If `wipeFilesystem` is set to false, Ignition will then attempt to reuse the exi
 
 When resolving paths, Ignition follows symlinks on all but the last element of a path. This ensures existing symlinks on a filesystem can be overwritten while still following symlinks as expected. When writing files, links, or directories, Ignition does not allow following symlinks outside the specified filesystem. When writing files, links, or directories on the `root` filesystem, Ignition follows symlinks as if it were executing in that root; a symlink to `/etc` is followed to `/etc` on the `root` filesystem. When writing files, links, or directories to any other filesystem, Ignition fails if it tries to follow a symlink outside that filesystem.
 
+## Making changes to `/proc`, `/sys`, `/dev`, `/tmp` or `/run`
+
+To create files, directories or symlinks in `/proc`, `/sys` or `/dev`, you should use [sysctl.d config files](https://www.mankier.com/5/sysctl.d) or [udev rules](https://www.mankier.com/7/udev).
+
+Similarly, to makes changes under the `/tmp` or `/run` paths, you should use [tmpfiles.d config files](https://www.mankier.com/5/tmpfiles.d).
+
+These directories are managed by system services (sysctl, udev, and systemd). Changes under the directories must come from the services' configuration. Otherwise, the services are not aware of the changes, and may ignore or overwrite them.
```

Comment:
```suggestion
Those directories are managed by system services (systemd, udev, etc.). Changes under those directories should come from the services' configuration. Otherwise, those services will not be aware of those changes, and may ignore or overwrite them.
```